tracer/test/Datadog.Trace.TestHelpers/MockLogsIntake.cs

```diff
@@ -248,7 +249,8 @@ private string Version2
             public static List<Log> DeserializeFromStream(Stream stream)
             {
                 using var sr = new StreamReader(stream);
-                using var jsonTextReader = new JsonTextReader(sr);
+                var json = sr.ReadToEnd();
+                using var jsonTextReader = new JsonTextReader(new StringReader(json));
```

Comment:
Why do we need to read this all in to memory? The existing code should work? 🤔

Reply:
I was getting a lot of `HttpListenerException` with connection close with the previous code, I guess it's a `HttpListener` thing.

Reply:
Forget it [b742ed6](https://github.com/DataDog/dd-trace-dotnet/pull/2531/commits/b742ed609e824d8aff6119cbfb611478be8f9410) I was missing to set the CIVisibility Logs environment variable to handle the flush correctly. I reverted the Mock changes.